_apps/first-timers-bot.md

```diff
@@ -0,0 +1,88 @@
+---
+title: First Timers Bot
+description: A friendly bot that helps with onboarding new Open Source Contributors
+slug: first-timers-bot
+screenshots:
+- https://raw.githubusercontent.com/hoodiehq/first-timers-bot/master/assets/Issue-Done.png
+author: agonzalez0515 ArlenePerez
```

Comment:
Based on the changes in https://github.com/probot/probot.github.io/pull/46 this should be able to become `authors: [ githubUserName, githubUserName ]`